style.css

```diff
@@ -546,8 +545,66 @@ button {
   transition: all 1s ease 0.5s;
 }
 
-.nav_wrapper {
+.header_wrapper {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 3rem;
+  align-content: center;
+
+  /* margin-left: 15px;
+  margin-right: 15px; */
+}
```

Comment:
- [ ] Kindly avoid leaving commented pieces of code in your repo as it makes your  codebase unclean :+1:

Reply:
removed uncommented codes